plugins/auth/local/index.js

```diff
@@ -195,13 +195,19 @@ LocalAuth.prototype.registerUser = function (req, res, next) {
 
 };
 
-LocalAuth.prototype.internalRegisterUser = function (user, cb) {
-  if (!user.email || !user.password || !user.retypePassword) {
+LocalAuth.prototype.internalRegisterUser = function(retypePasswordRequired, user, cb) {
+  if (!user.email || !user.password) {
     return cb(new auth.errors.UserRegistrationError('email, password and retyped password are required!'));
```

Comment:
Just to note this error message could now be misleading when there's no retyped password field:

![retyped](https://user-images.githubusercontent.com/922987/28668462-cd11d92e-72c7-11e7-97ba-8165377e4b59.png)